src/platforms/hosted/jlink.c

```diff
@@ -317,6 +325,24 @@ static bool jlink_get_version(void)
 	return true;
 }
 
+static bool jlink_get_extended_capabilities(void)
+{
+	uint8_t buffer[32U];
+
+	if (!jlink_simple_query(JLINK_CMD_INFO_GET_PROBE_EXTENDED_CAPABILITIES, buffer, sizeof(buffer)))
+		return false;
+
+	uint32_t ext_caps[4];
```

Comment:
Let's store this in the `jlink_s` struct already, even if we don't need it right now. Make the `jlink_s->capabilities` bitfield an array of 4 `uint32_t`, the first 32 bits should be shared between the regular capabilities and extended capabilities responses

Reply:
Okay, I'll put an array there. Reading RM08001 seems to suggest that `extended_capabilities[0]` is an exact copy of `capabilities` value save for most significant bit (31), which indicates there ARE extended capabilities.
Am I to correct the 9 references to that single uint32_t to refer to `capabilities[0]` now? It would be easier (on the existing code) to store the remainder of ext_caps in a separate uint32_t[3] array.

Reply:
Yes, currently there is no check for any extended capability.
I'm not against keeping the extended capabilities sepperate

Reply:
Given the docs say they should be identical save for that MSb, we'd prefer them to use the same storage, and extend the storage, rather than using new storage. It'll keep the J-Link structure simpler.